plugin/plugins.go

```diff
@@ -69,7 +70,8 @@ func NewHandlerFunc(ops []reflect.Type, events []reflect.Type, listenOp func(op
 // WriteOp writes the given op into the op.Ops queue.
 func WriteOp(op *op.Ops, c any) {
 	defer clip.Rect{}.Push(op).Pop()
-	pointer.InputOp{Tag: c}.Add(op)
+	// pointer.GrabCmd{Tag: c}.Add(op)//todo
+	pointer.GrabCmd{Tag: c}.ImplementsCommand()
```

Comment:
I encode custom `ops` as "Tag" in `InputOp`. It was a hack, I think it's no longer necessary, since the `Events` seems to accept the `Tag` and `Filter`. So we can match both of them, if we replace the original Gio (unsafe hack).